.github/workflows/build_and_test.yaml

```diff
@@ -10,6 +10,9 @@ on:
   push:
     branches:
       - main
+  schedule:
+    # Run every morning to detect flakiness and broken dependencies
+    - cron: '43 1 * * *'
```

Comment:
Is there a benefit to running this often or is once a week good enough?

Reply:
Once a week is probably good enough.

I just took these two lines from another PickNik workflow yaml that I had in my workspace